geaflow/geaflow-deploy/geaflow-on-ray-community/src/main/java/com/antgroup/geaflow/cluster/ray/clustermanager/RayClient.java

```diff
@@ -41,6 +42,7 @@ public static ActorHandle<RayMasterRunner> createMaster(ClusterConfig clusterCon
         ActorHandle<RayMasterRunner> masterRayActor = Ray
             .actor(RayMasterRunner::new, clusterConfig.getConfig())
             .setMaxRestarts(clusterConfig.getMaxRestarts())
+            .setLifetime(ActorLifetime.DETACHED)
```

Comment:
"detached" actor will not be killed when the ray job is finished or failed. Does it make sense for your scenes? For more detail, you can read this doc https://docs.ray.io/en/latest/ray-core/actors/named-actors.html?highlight=detached#actor-lifetimes

Reply:
it make sense for out scenes. we expected the actor run with long-running mode.